prover/src/constraints/evaluator/default.rs

```diff
@@ -237,7 +237,7 @@ where
     ) {
         // initialize buffers to hold trace values and evaluation results at each step
         let mut main_frame = EvaluationFrame::new(trace.trace_info().main_trace_width());
-        let mut aux_frame = EvaluationFrame::new(trace.trace_info().aux_trace_width());
+        let mut aux_frame = EvaluationFrame::new(trace.trace_info().aux_segment_width());
```

Comment:
Looking at things like this, I'm wondering if we should use `aux_trace` everywhere rather than `aux_segment`. This would be more consistent with how we name main trace segment. Basically, we have `main_trace` and `aux_trace`. And so things like `aux_segment_commitment` become `aux_trace_commitment` etc.